tests/integration-tests/tests/ultraserver/test_gb200.py

```diff
@@ -281,10 +363,46 @@ def test_gb200(
     )
     cluster = clusters_factory(cluster_config)
 
+    # Test IMEX and topology configuration for queue with IMEX support
     assert_imex_healthy(cluster, queue_with_imex, compute_resource_with_imex, max_queue_size)
+    assert_topology_plugin_configured(
+        cluster, queue_with_imex, compute_resource_with_imex, f"{max_queue_size}", max_queue_size
+    )
+
+    # Test that IMEX and topology are not configured for queue without IMEX support
+    assert_imex_not_configured(cluster, queue_without_imex, compute_resource_without_imex)
+    assert_topology_plugin_not_configured_for_queue(cluster, queue_without_imex, compute_resource_without_imex)
+
+    # Test cluster update with changed topology configuration
+    max_queue_size_updated = 3
+    updated_cluster_config = pcluster_config_reader(
+        config_file="pcluster.config.update.yaml",
+        bucket_name=bucket_name,
+        head_node_start_script=headnode_start_filename,
+        max_queue_size=max_queue_size_updated,
+        queue_with_imex=queue_with_imex,
+        compute_resource_with_imex=compute_resource_with_imex,
+        queue_without_imex=queue_without_imex,
+        compute_resource_without_imex=compute_resource_without_imex,
+    )
 
-    # IMEX is not configured on compute resource that does not support it
+    cluster.stop()
+    wait_for_computefleet_changed(cluster, "STOPPED")
+    cluster.update(str(updated_cluster_config), force_update=True)
+    cluster.start()
+    wait_for_computefleet_changed(cluster, "RUNNING")
+    # Wait for compute nodes to be fully running
+    wait_for_instances_in_compute_resource(
+        cluster, queue_with_imex, compute_resource_with_imex, ["running"], max_queue_size_updated
+    )
+
+    # Verify imex and topology plugin configuration after update
+    assert_imex_healthy(cluster, queue_with_imex, compute_resource_with_imex, max_queue_size_updated)
+    assert_topology_plugin_configured(
+        cluster, queue_with_imex, compute_resource_with_imex, f"{max_queue_size_updated}", max_queue_size_updated
```

Comment:
Non-blocking Nitpick . We know the max_queue_size_updated is the BlockSize in string so its not necessary to send both.